index.html

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title>JS Onboard Project</title>
-	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
-</head>
-
-<body>
-	<p>Hello</p>
-</body>
-
+  <head>
+    <title>JS Onboard Project</title>
+    <script
+      type="text/javascript"
+      charset="utf-8"
+      src="third_party/jquery-2.0.3.min.js"
+    ></script>
```

Comment:
Rather keep this to one line.
```suggestion
<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
```

Reply:
Bump